package.json

```diff
@@ -67,6 +67,7 @@
     "npm-run-all": "4.1.5",
     "npm-run-scripts": "2.1.2",
     "prettier": "2.3.1",
+    "shx": "^0.3.3",
```

Comment:
For maximum consistency (and security, in a sense), we should not use variable versioning ("^").